### ISSUE TITLE
Issue with getting the results to show

### ISSUE THREAD

**Issue by normalusername1** [kampta/face-seg#3]
I created a model but while running the test.py file, all it does is print 
`12  files in folder  /content/drive/My Drive/data (1)/raw/samples
Figure(640x480)`

the plt.show() doesn't actually show anything. I know it's not a problem with my ide or matplotlib because I tried plt.show() with a different example and it worked fine. The code I'm using to test it is `!python '/content/face-seg/test.py' --data-folder '/content/drive/My Drive/data (1)/raw/samples' --pre-trained '/content/scratch/249/models/best.pt'`

**Comment by kampta** [comment]
Probably an issue with `plt.show`. Try modifying the code a bit to dump images on disk instead. You may try `plt.savefig`

**Comment by normalusername1** [comment]
Ok plt.savefig() works fine. I have another question though, why does the end result get shrunk so much? Is it possible for me to make the end mask the same size as the input picture?

**Comment by kampta** [comment]
Default model architecture (mobilenet) expects 224x224 input I believe. Input images are resized to this size and output masks will also come out of this size (check `dataset.py`). You can resize the final output masks to whichever size you want by interpolation (using Pillow's `Image.resize()` or torchvision's `Resize` or whichever library you like).

You can also pick a different neural network architecture, or modify the current one to work with bigger size images.
Closing the issue for now.
